packages/label-sync/src/labels.json

```diff
@@ -55,14 +55,19 @@
       "description": "Request for information or clarification. Not an issue.",
       "color": "c5def5"
     },
+    {
+      "name": "priority: p3",
+      "description": "Desirable enhancement or fix. May not be include in next release.",
```

Comment:
```suggestion
      "description": "Desirable enhancement or fix. May not be included in next release.",
```

Reply:
oops sorry, missed this. I'll create a typo fix

Reply:
sgtm. thanks!